### PR TITLE
no sig standardization in cw 45 due to OSBA event

### PR DESCRIPTION
**Pull request by fkr**
Signed-off-by: Felix Kronlage-Dammers <fkr@hazardous.org>

### CODE REVIEW

**Review by itrich — APPROVED** [review]
LGTM!

**Comment by garloff** [comment]
Thanks!

File: team_meetings.yml [hunk 1]
```diff
@@ -224,6 +224,8 @@ events:
       interval:
         weeks: 2
       until: 2022-12-31
+      except_on:
+        - 2022-11-10 14:05:00
```
Comment (garloff) [inline team_meetings.yml:228]: Indeed, thanks!

**Review by garloff — APPROVED** [review]
/lgtm

**Pull request merged**